controllers/volumereplicationclass.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-func (r VolumeReplicationReconciler) getVolumeReplicaClass(logger logr.Logger, vrcName string) (*replicationv1alpha1.VolumeReplicationClass, error) {
+func (r VolumeReplicationReconciler) getVolumeReplicationClass(logger logr.Logger, vrcName string) (*replicationv1alpha1.VolumeReplicationClass, error) {
```

Comment:
add source code comment for all the functions wherever it's missing?

Reply:
I have added for this function, rest can be a followup PR.

Reply:
IMO its good to have everything in a single PR instead  fixing it bit by bit in followup PR's